cri/ocicni/types.go

```diff
@@ -0,0 +1,24 @@
+package ocicni
```

Comment:
I do not think defining an interface should use a filename of `types.go`.
Instead, maybe cnimanager.go is a better one. WDYT?

Reply:
`cnimanager.go` doesn't make sense, there is a file named `cni_manager.go`. So maybe `apis.go` or something else?

Reply:
@YaoZengzeng 
Any advice?

Reply:
I think `types.go` is fine, we could expand it if we need more types 😄